src/it/java/SmbjTest.java

```diff
@@ -333,6 +333,15 @@ public void testRpc() throws IOException, SMBApiException, URISyntaxException {
     }
 
 
+    @Test
+    public void testSID() {
```

Comment:
Could you move this test to the `SIDTest.groovy` class? Also please don't println, just assert :)